Dockerfile

```diff
@@ -0,0 +1,22 @@
+# We need a lightweight OS and Node preinstalled
+FROM node:9.11-alpine
+MAINTAINER stockerl@unternehmertum.de
+
+RUN npm install webpack webpack-cli
+
+RUN mkdir -p /var/www/dpschool
+WORKDIR /var/www/dpschool
+
+# Install root deps
+COPY ./package.json package.json
```

Comment:
do we need that? I think this might be copied already above with mkdir -p /var/www/dpschool. Can we double check?

Reply:
Why should it be there? mkdir creates the given (empty) dir. -p creates parent dirs, if needed. 

Nevertheless, I doublechecked it
```
Step 6/13 : RUN ls -al
 ---> Running in 5b184d2e80c3
total 8
drwxr-xr-x    2 root     root          4096 Jun 29 19:25 .
drwxr-xr-x    3 root     root          4096 Jun 29 19:25 ..
```

Reply:
sorry, bad explanation from my side:

The line: COPY . . should copy the whole dir? After that we can run the appropriate yarn installs and therefore delete the 2x COPY ./package.json package.json? Would that work?

Reply:
I'd create a ticket for that and focus on finishing the website first. What do you think?

Reply:
sure, I wanted to make sure if I understand the setup